### ISSUE TITLE
Automated backups

### ISSUE THREAD

**Issue by janlucaklees** [kannji/api-server#6]
Automated backups of the user and dictionary data are essentially. 

- [x] Small backup scripts for the individual docker containers (API-Server, Nginx)
- [ ] One script to bundle all individual backups together into one big backup
- [ ] An option to restore backups.
- [ ] Send backups to remote location

**Comment by 4kobras** [comment]
Maybe use a versioning system internally? A private Git server with a nice, slim frontend might just do the trick. *insert thinking emoji here*.

**Comment by janlucaklees** [comment]
Yeah I also thought about something like that. We may not want to make full bakups every time. Some incremental system would be nice, with complete backups every weeks or so.

**Comment by 4kobras** [comment]
Sounds like something svn and a cronjob could do perfectly. If I recall correctly (*iirc*, for future reference), Subversion stores only the differences between versions. However, while it does so, Git is more efficient in storage and performance while storing images in its commits. There are other versioning tools, but I doubt there are any that are that well documented and supported by other tools as SVN and Git.
